### PR TITLE
Added e2e tests for `member.edited` webhook

### PR DESCRIPTION
**Pull request by illiteratewriter**
#15537

### CODE REVIEW

File: ghost/core/test/e2e-webhooks/members.test.js [hunk 1]
```diff
@@ -121,4 +121,55 @@ describe('member.* events', function () {
...
+                }]
+            })
+            .expectStatus(201);
+        
+        const id = res.body.members[0].id;
+
+        const updatedMember = {...res.body.members[0]};
+        updatedMember.name = 'Ghost';
+        updatedMember.email = 'ghost@example.com';
+        updatedMember.note = 'ghost note';
```
Comment (naz) [inline ghost/core/test/e2e-webhooks/members.test.js:149]: Not something to change here necessarily, just small piece of feedback to improve future contributions. It's generally a more readable and maintainable solution when tests can achieve the needed state in as few steps as possible (unless it's some hacky shortcut that is vague). In this case, just editing the name would have triggered the 'member.edited' event. Less code - better code 😊

**Review by naz — APPROVED** [review]
Thank you @illiteratewriter! Another great contribution 👍

**Comment by illiteratewriter** [comment]
Thank you @naz. It feels really nice contributing to Ghost.

**Comment by ErisDS** [comment]
Hi again @illiteratewriter 👋 this PR is now merged ready for today's release. I'll take a look at the custom theme settings import PR again in a bit.

Thanks for all your continued contributions, we're loving it ❤️

**Comment by illiteratewriter** [comment]
@ErisDS It's really great working on Ghost. 

I'd like to look into the [i18n Milestone 2](https://github.com/TryGhost/Ghost/issues/15502) if it's okay.

**Comment by ErisDS** [comment]
That would be incredible - we've posted it in hacktoberfest and on dev.to but not got any interest yet! If you have any questions post them on the issue and I'll get back to you as fast as I can!

**Pull request merged**